cdap-credential-ext-gcp-wi/src/test/java/io/cdap/cdap/security/spi/credential/GcpWorkloadIdentityCredentialProviderTest.java

```diff
@@ -16,6 +16,9 @@
 
 package io.cdap.cdap.security.spi.credential;
 
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
```

Comment:
nit: It would be good to keep it consistent throughout the class, whether we want to use `Mockito.mock` or `mock`. Currently with this change it is becoming scattered.

Reply:
Done, thanks!